server/src/main/java/chat/engine/Connect.java

```diff
@@ -63,7 +65,11 @@ public void run() {
             try {
                 String message = reader.readLine();
 
-                sendMessage(message);
+                if (message.contains(PM)) {
```

Comment:
it is better to move command check to another method, or class. In future you can obtain much more command for various action, and this method will grow